blockchain/blockdao/blob_store.go

```diff
@@ -22,4 +35,173 @@ type (
 		GetBlobsByHeight(uint64) ([]*types.BlobTxSidecar, []string, error)
 		PutBlock(*block.Block) error
 	}
+
+	// storage for past N-day's blobs, structured as blow:
+	// 1. Storage is divided by hour, with each hour's data stored in a bucket.
```

Comment:
I think the design with multiple buckets is a bit complicated, Why we need this? If we store the latest-N blobs directly in a single bucket, it will look more simple as following

```
blobStore struct {
    limit uint64
    base, tip uint64
    kvstore db.KVStore
}
```

When ```tip - base > limit```, we can simply clean up the `base`.

Reply:
changed to a ring-buffer implementation